jsx/InfoPanel.tsx

```diff
@@ -1,12 +1,13 @@
-import PropTypes from 'prop-types';
+import {ReactNode} from 'react';
 
 /**
  * Display a message in an information panel.
  *
  * @param {object} props - React props
- * @return {JSX}
+
+ * @returns {ReactNode} - the InfoPanel
  */
-function InfoPanel(props) {
+function InfoPanel(props: {children: ReactNode}): ReactNode {
```

Comment:
function InfoPanel : React.FC<{children: ReactNode}>

Reply:
Use generics to inherit parent type definitions.

Reply:
Making use of the React.FC generic type will specifies the return type. But it seems it will be deprecated soon, so let's leave it as is: https://github.com/facebook/create-react-app/pull/8177